deepchecks/checks/performance/simple_model_comparison.py

```diff
@@ -127,90 +180,89 @@ def _find_score(self, train_ds: Dataset, test_ds: Dataset, task_type: ModelType,
             else:
                 raise DeepchecksValueError(f'Unknown task type - {task_type}')
 
-            clf = clf.fit(x_train, y_train)
-            simple_pred = clf.predict(x_test)
-
+            clf.fit(x_train, y_train)
+            return ScaledModel(clf, sn)
         else:
             raise DeepchecksValueError(
                 f'Unknown model type - {self.simple_model_type}, expected to be one of '
                 f"['random', 'constant', 'tree'] "
                 f"but instead got {self.simple_model_type}"  # pylint: disable=inconsistent-quotes
             )
 
-        y_test = test_ds.label_col.values
-
-        scorer = get_scorer_single(model, train_ds, self.scorer)
-
-        simple_score = scorer(_DummyModel, Dataset(pd.DataFrame(simple_pred), label=y_test))
-        pred_score = scorer(model, Dataset(test_ds.features_columns, label=y_test, cat_features=test_ds.cat_features))
-
-        return simple_score, pred_score, scorer.name
-
-    def _simple_model_comparison(self, train_dataset: Dataset, test_dataset: Dataset, model):
-        Dataset.validate_dataset(train_dataset)
-        Dataset.validate_dataset(test_dataset)
-        train_dataset.validate_label()
-        test_dataset.validate_label()
-        validate_model(test_dataset, model)
-
-        task_type = task_type_check(model, train_dataset)
-        simple_score, pred_score, score_name = self._find_score(train_dataset, test_dataset, task_type, model)
-
-        if score_name == DEFAULT_SINGLE_SCORER[task_type]:
-            score_name = str(score_name) + ' (Default)'
-
-        ratio = get_scores_ratio(simple_score, pred_score, self.maximum_ratio)
-
-        text = f'The given model performance is {_more_than_prefix_adder(ratio, self.maximum_ratio)} times the ' \
-               f'performance of the simple model, measuring performance using the {score_name} metric.<br>' \
-               f'{type(model).__name__} model prediction has achieved a score of {format_number(pred_score)} ' \
-               f'compared to Simple {self.simple_model_type} prediction ' \
-               f'which achieved a score of {format_number(simple_score)} on tested data.'
-
-        models = [f'Simple model - {self.simple_model_type}', f'{type(model).__name__} model']
-        results = [simple_score, pred_score]
-        fig = go.Figure([go.Bar(x=models, y=results)])
-        fig.update_layout(width=600, height=500)
-        fig.update_yaxes(title=score_name)
-
-        return CheckResult({'given_model_score': pred_score,
-                            'simple_model_score': simple_score,
-                            'ratio': ratio},
-                           display=[text, fig])
-
-    def add_condition_ratio_not_less_than(self, min_allowed_ratio: float = 1.1):
+    def add_condition_ratio_not_less_than(self, min_allowed_ratio: float = 1.1, classes: List[Hashable] = None):
```

Comment:
Let me know what you think, but it may be better to set this by default only on a single metric (e,g. f1)

Reply:
it's not possible to set a default since you don't know the model type